lua/gitpad/init.lua

```diff
@@ -5,11 +5,19 @@ local gitpad_win_id = nil
 
 M.config = {
   title = 'gitpad',
-  border = 'single',
   dir = vim.fs.normalize(vim.fn.stdpath('data') .. '/gitpad'),
-  style = '',
   default_text = nil,
   on_attach = nil,
+  window_type = 'floating',
+  floating_win_opts = {
```

Comment:
I'm not sure if `floating_win_opts` would be appropriate since this PR will make it support floating and non-floating window.

Probably `win_opts` would suffice?

Reply:
Oh nvm. You also added `split_win_opts` below 🤦